api/stream/design.go

```diff
@@ -191,7 +191,7 @@ var _ = Service("stream", func() {
 	Method("UpdateSource", func() {
 		Meta("openapi:summary", "Update source")
 		Description("Update the source.")
-		Result(Source)
+		Result(Task)
 		Payload(UpdateSourceRequest)
```

Comment:
Forward compatibility, there is no blocking operation when updating `HTTP Source`, but different source types may require a long-running operation.